src/components/calcite-pick-list-item/calcite-pick-list-item.tsx

```diff
@@ -270,6 +278,43 @@ export class CalcitePickListItem {
     ) : null;
   }
 
+  renderContent(): VNode {
+    const { description, label, el } = this;
+    const hasContentSlot = getSlotted(el, SLOTS.content);
+
+    return hasContentSlot ? (
+      <div
+        class={CSS.content}
+        onClick={this.pickListClickHandler}
+        onKeyDown={this.pickListKeyDownHandler}
+        ref={(focusDiv): HTMLDivElement => (this.focusDiv = focusDiv)}
+        tabIndex={0}
+      >
+        <div onClick={this.contentSlotClickHandler}>
+          <slot name={SLOTS.content} />
+        </div>
+      </div>
+    ) : (
+      <label
+        aria-label={label}
+        class={CSS.label}
+        onClick={this.pickListClickHandler}
+        onKeyDown={this.pickListKeyDownHandler}
+        ref={(focusEl): HTMLLabelElement => (this.focusEl = focusEl)}
+        tabIndex={0}
+      >
+        <div
+          aria-checked={this.selected.toString()}
```

Comment:
Is this `role` and `aria-checked` no longer needed when the content slot is used? Should they be moved onto the div surrounding the content slot?